maitake-sync/README.md

```diff
@@ -157,6 +157,49 @@ critical section, are described [here][interrupt-cfgs].
 [single-core]: https://docs.rs/portable-atomic/latest/portable_atomic/#optional-cfg
 [interrupt-cfgs]: https://github.com/taiki-e/portable-atomic/blob/HEAD/src/imp/interrupt/README.md
 
+### overriding blocking mutex implementations
+
+In addition to async locks, `maitake-sync` also provides a [`blocking`] module,
```

Comment:
Link to default mutex here?

Reply:
There is a mention of `DefaultMutex` on the next paragraph --- do you think
that's sufficient?

Reply:
Works for me!